src/main/resources/application.yaml

```diff
@@ -59,3 +61,23 @@ white-label:
     title: Zeebe Simple Monitor
     css.path: css/custom.css
     js.path: js/custom.js
+---
+spring:
+  config:
+    activate:
+      on-profile: kafka
+  datasource:
+    url: ${POSTGRES_DATASOURCE_URL:jdbc:postgresql://localhost:5432/postgres}
+    username: ${POSTGRES_DATABASE_USER:postgres}
+    password: ${POSTGRES_DATABASE_PASSWORD:temp}
+    driverClassName: org.postgresql.Driver
+  jpa:
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
+    hibernate:
+      ddl-auto: update
```

Comment:
ah this might cause some issues with how the DB is configured in the chart already. See https://github.com/elisapolystar/zeebe-simple-monitor-chart/blob/main/templates/deployment.yaml#L53 for how it's done and https://github.com/elisapolystar/deployment-sponde/blob/ca90c6a97f1202a46872656fb8c59b7a6c1f3594/apps/values.yaml#L429 for an example. I would suggest putting these Postgres specific configs in a new profile block or follow how DBs are set in the zeebe-simple-monitor-chart

Reply:
Resolved